src/main/java/com/cloudogu/plantuml/PlantUmlResource.java

```diff
@@ -51,9 +52,13 @@ public Response createSvg(@PathParam("content") String encodedContent) throws IO
     final TranscoderSmart transcoder = new TranscoderSmart();
     final String decode = transcoder.decode(encodedContent);
 
-    SourceStringReader reader = new SourceStringReader(decode);
-    final StreamingOutput streamingOutput = outputStream -> reader.generateImage(outputStream, new FileFormatOption(FileFormat.SVG));
-    return Response.ok(streamingOutput).lastModified(new Date()).header("Cache-Control", "public, max-age=31536000").build();
+    try {
+      SourceStringReader reader = new SourceStringReader(decode);
+      final StreamingOutput streamingOutput = outputStream -> reader.generateImage(outputStream, new FileFormatOption(FileFormat.SVG));
+      return Response.ok(streamingOutput).lastModified(new Date()).header("Cache-Control", "public, max-age=31536000").build();
+    } catch (Exception e) {
+      throw new PlantUMLRenderException();
```

Comment:
Please keep the root cause in the `PlantUMLRenderException`, so we can see the whole exception in the log.